src/aks-preview/azext_aks_preview/_consts.py

```diff
@@ -86,6 +86,7 @@
 # consts for addons
 # http application routing
 CONST_HTTP_APPLICATION_ROUTING_ADDON_NAME = "httpApplicationRouting"
+CONST_WEB_APPLICATION_ROUTING_ADDON_NAME = "webApplicationRouting"
```

Comment:
The const declared here is intended to be used as the key of the addon_profiles in mc and its corresponding value is a ManagedClusterAddonProfile model.

However, this addon is using a separate profile. We may remove the const declaration.